pkg/zcmd/zfs/get/builder.go

```diff
@@ -169,37 +168,37 @@ func (v *VolumeGetProperty) Build() (*VolumeGetProperty, error) {
 	v = v.Validate()
 	v.appendCommand(&c, bin.ZFS)
 
-	v.appendCommand(&c, fmt.Sprintf(" %s ", Operation))
+	v.appendCommand(&c, " "+Operation+" ")
 
 	if IsScriptedModeSet()(v) {
-		v.appendCommand(&c, fmt.Sprintf(" -H "))
+		v.appendCommand(&c, " -H ")
 	}
 
 	if IsParsableModeSet()(v) {
-		v.appendCommand(&c, fmt.Sprintf(" -p "))
+		v.appendCommand(&c, " -p ")
 	}
 
 	if IsFieldListSet()(v) {
-		v.appendCommand(&c, fmt.Sprintf(" -o "))
+		v.appendCommand(&c, " -o ")
 		for _, f := range v.FieldList {
-			v.appendCommand(&c, fmt.Sprintf("%s,", f))
+			v.appendCommand(&c, ","+f)
```

Comment:
@asquare14 
I think here it would be v.appendCommand(&c, f+",")
Am I right @mittachaitu ?

Reply:
Yes, correct it should be the other way around